packages/shenanigans-manager/setup/package.json

```diff
@@ -13,12 +13,12 @@
         "sinon-chai": "^3.7.0"
     },
     "scripts": {
-        "clean": "rm -rf lib *.tsbuildinfo",
+        "clean": "rm -rf dist lib *.tsbuildinfo",
         "hydrate": "yarn shenanigans-manager hydrate",
         "link": "yarn link",
         "publish:ci": "yarn shenanigans-manager publish-if-updated",
         "test": "yarn run test:setup && yarn run test:run",
-        "test:run": "yarn mocha-headless-chrome --file test/index.html",
+        "test:run": "yarn mocha-headless-chrome --file test/index.html --polling 250",
```

Comment:
> Adds `--polling 250` to tests because `mocha-headless-chrome`'s Puppeteer `page.waitForFunction` call was trying to evaluate `window.__mochaResult__` only before tests finish, and not after (yes, this took way too long to figure out)

I also tried `--polling mutation` and while that fixed it locally, CI was still too slow.